scripts/prep_mintpy.py

```diff
@@ -0,0 +1,552 @@
+from dolphin.workflows.config import OPERA_DATASET_ROOT
+from mintpy.utils import arg_utils, isce_utils, ptime, readfile, writefile
+
+####################################################################################
+EXAMPLE = """example:
+
+  python3 ./prep_sweets.py -u 'interferograms/stitched/*.unw'  -m gslcs/t064_135516_iw3/20190704/static_layers_t064_135516_iw3.h5
+
+  ## example commands after prep_sweets.py
+  reference_point.py timeseries.h5 -y 500 -x 1150
+  generate_mask.py temporalCoherence.h5 -m 0.7 -o maskTempCoh.h5
+  tropo_pyaps3.py -f timeseries.h5 -g inputs/geometryRadar.h5
+  remove_ramp.py timeseries_ERA5.h5 -m maskTempCoh.h5 -s linear
+  dem_error.py timeseries_ERA5_ramp.h5 -g inputs/geometryRadar.h5
+  timeseries2velocity.py timeseries_ERA5_ramp_demErr.h5
+  geocode.py velocity.h5 -l inputs/geometryRadar.h5
+"""  # noqa: E501
+
+# """
+# Scott TODO:
+# - dont need the ifg sample file, the unwrapped files should have everything
+# - UTM_ZONE, EPSG from the stitched IFG (it won't work to get a single GSLC burst)
+# - pixel size is wrong since we're taking range/azimuth size, instead of geocoded size
+# - HEIGHT: do we wanna try to get that from the saved orbit info?
+# -  cor_files  = [x.split(".unw")[0] + ".cor" for x in unw_files]
+# -    This might be wrong for dolphin? to verify
+
+# Scott did:
+# - remove bbox
+# - "set all data less than 0" thats a fringe specific thing to do -1, -2 for tcoh
+
+# """
+
+
+def _create_parser():
+    parser = argparse.ArgumentParser(
+        description="Prepare SWEETS products for MintPy",
+        formatter_class=argparse.RawTextHelpFormatter,
+        epilog=EXAMPLE,
+    )
+
+    parser.add_argument(
+        "-u",
+        "--unw-file-glob",
+        type=str,
+        default="./interferograms/unwrapped/*.unw.tif",
+        help="path pattern of unwrapped interferograms (default: %(default)s).",
+    )
+    parser.add_argument(
+        "-c",
+        "--cor-file-glob",
+        type=str,
+        default="./interferograms/stitched/*.cor",
+        help="path pattern of unwrapped interferograms (default: %(default)s).",
+    )
+    parser.add_argument(
+        "-g",
+        "--geom-dir",
+        default="./geometry",
+        help="Geometry directory (default: %(default)s).\n",
+    )
+    parser.add_argument(
+        "-m",
+        "--meta-file",
+        dest="metaFile",
+        type=str,
+        help="GSLC metadata file or directory",
+    )
+    parser.add_argument(
+        "-b",
+        "--baseline-dir",
+        dest="baselineDir",
+        type=str,
+        default=None,
+        help="baseline directory (default: %(default)s).",
+    )
+    parser.add_argument(
+        "-o",
+        "--out-dir",
+        dest="outDir",
+        type=str,
+        default="./mintpy",
+        help="output directory (default: %(default)s).",
+    )
+    parser.add_argument(
+        "-r",
+        "--range",
+        dest="lks_x",
```

Comment:
we should figure out if we want 
1. the actual number of looks we took on the CSLCs
2. the effective number of radar looks, accounting for the difference in the range/az. resolution of the SLCs and the geocoded SLCs